roles/sushy_emulator/tasks/apply_resources.yml

```diff
@@ -0,0 +1,41 @@
+- name: Create the sushy-emulator namespace
+  kubernetes.core.k8s:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit)}}"
+    context: "{{ cifmw_openshift_context | default(omit) }}"
+    name: "{{ cifmw_sushy_emulator_namespace }}"
+    kind: Namespace
+    state: present
+
+- name: Apply Sushy Emulator resources
+  kubernetes.core.k8s:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit)}}"
+    context: "{{ cifmw_openshift_context | default(omit) }}"
+    name: "{{ cifmw_sushy_emulator_namespace }}"
+    state: present
+    src: "{{ (cifmw_sushy_emulator_resource_directory, item+'.yaml') | path_join }}"
+  with_items:
+    - secret
+    - configmap
+    - service
+    - route
+
+- name: Apply Sushy Emulator pod resource
+  kubernetes.core.k8s:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit)}}"
+    context: "{{ cifmw_openshift_context | default(omit) }}"
+    name: "{{ cifmw_sushy_emulator_namespace }}"
+    state: present
+    src: "{{ (cifmw_sushy_emulator_resource_directory, 'pod.yaml') | path_join }}"
+    wait: yes  # noqa: yaml[truthy]
+    wait_sleep: 10
+    wait_timeout: 360
+    wait_condition:
+      type: Ready
+      status: "True"
+
+- name: Set sushy url for ocp installation
```

Comment:
This task seems out of place, given that `_ingress_domain` is created in another task file